teams/vsa/engineering/forms-system/custom_component_multiple.md

```diff
@@ -0,0 +1,138 @@
+# Extending RJSF Behavior Using a Custom React Component
+
+(Based mostly on [bradl3yC](https://github.com/bradl3yC)'s write-up of his solution)
+
+## The Problem:
+Using a custom component requires that you pass in a schema for that component.  It essentially maps a small slice of form state to the component.  Because the same component is used twice but we are passing in a different schema, each of those components, each of those only has access to its own individual schema/form state. (Read [Background: The Original Problem](#background-the-original-problem) for further details.)
```

Comment:
I believe the part `, each of those components, ` is unnecessary here and makes this a bit confusing to read

Reply:
thanks, that was a weird typo